processes/procstat.go

```diff
@@ -76,18 +76,18 @@ func (psc *procStatsCollector) GetStats(procPath string) (map[string][]Proc, err
 			fstat := filepath.Join(procPath, file.Name(), procStat)
 			procStat, err := ioutil.ReadFile(fstat)
 			if err != nil {
-				return nil, err
+				continue
```

Comment:
Consider adding logs in places where you put "continue" to inform what happened. Those logs should be captured by the newest snap.